addons/account/models/chart_template.py

```diff
@@ -391,6 +408,21 @@ def _pre_load_data(self, template_code, company, template_data, data):
             if model in data:
                 data[model] = data.pop(model)
 
+        # Translate the untranslatable fields we want to translate anyway
+        untranslatable_model_fields = self._get_untranslatable_fields_to_translate()
+        untranslatable_target_lang = self._get_untranslatable_fields_target_language(template_code, company)
```

Comment:
Thanks. I guess this is resolved since it is r+?

It would also be a possibility. There may not be a language in the context though (see comment in `_get_untranslatable_fields_target_language`).
Taking the company language (if possible) or using `get_lang` makes it more robust imo (it also was discussed at some point).
Having it as a function also makes it a bit more flexible.